src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -44,8 +44,28 @@ const OPTIONS = [
   "device_id",
 ];
 
-const getType = (action: Action | undefined) =>
-  action ? OPTIONS.find((option) => option in action) : undefined;
+const getType = (action: Action | undefined) => {
```

Comment:
Why can't we re-use `getActionType` ?

Reply:
Because we would have to change all the element names 😅 but yes, I do want to do that, found it a bit late though

Reply:
aaah ok . yeah not this pr.